From aa925b4e0c9cf2835ffe6e0b4922da9cb9686495 Mon Sep 17 00:00:00 2001
From: Sebastien Guiriec <s-guiriec@ti.com>
Date: Tue, 5 Apr 2011 12:38:46 +0200
Subject: [PATCH 09/12] ASoC: ABE FW: Update firmware input/output scaling

Update ABE firmware in order to increase input/output scaling

Signed-off-by: Sebastien Guiriec <s-guiriec@ti.com>
---
 sound/soc/omap/abe/abe_firmware.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

Index: kernel/sound/soc/omap/abe/abe_firmware.c
===================================================================
--- kernel.orig/sound/soc/omap/abe/abe_firmware.c	2011-04-07 00:20:27.187130227 +0900
+++ kernel/sound/soc/omap/abe/abe_firmware.c	2011-04-07 15:14:16.568358393 +0900
@@ -55,7 +55,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  */
-0x00009250,  /* VERSION NUMBER */
+0x0000925E,  /* VERSION NUMBER */
 0x00002000,  /* PMEM LENGTH IN BYTES */
 0x00001404,  /* CMEM LENGTH IN BYTES */
 0x00010000,  /* DMEM LENGTH IN BYTES */
@@ -3380,8 +3380,8 @@
 0x0033fbe6,
 0x00dd3502,
 0x000fea26,
-0x003c0001,
-0x00080002,
+0x00400000,
+0x00100002,
 0x007f0001,
 0x00040002,
 0x00040002,
@@ -3545,7 +3545,7 @@
 0x00000000,
 0x00000000,
 0x00000000,
-0x00009250,
+0x0000925E,
 0x04000385,
 0x03CB040B,
 0x005103D5,
